src/sherwood.cr

```diff
@@ -95,6 +95,7 @@ class Sherwood
 
       # TODO: Variable Operations
       # TODO: Control Flow
+      when 0x70 then insp -= popType(SWInt, stack)
```

Comment:
I'm not sure I really understand the purpose of this opcode. What advantage does it give over `goto`, and why is it `-=` instead of `+=`?

Reply:
Idk why I did `-=` thats just me